scripts/remove_boilerplate.py

```diff
@@ -34,6 +35,7 @@
 
 IndexTuple = namedtuple('IndexTuple', ['index', 'domain', 'url', 'warc',
                                        'offset', 'length', 'status', 'mime'])
+whitelist = []
```

Comment:
You're not using a `list` for testing inclusion, are you? :fearful:

Reply:
Hát így már, hogy rákérdeztél, nem fogom :)